frontend/app.py

```diff
@@ -4,9 +4,22 @@
 
 @app.route('/')
 def index():
+    roadnet_file_path = request.args.get('roadnetFile')
+    log_file_path = request.args.get('logFile')
+    
+    if roadnet_file_path.startswith("/"):
+        roadnet_file_root_dir = "replay"
+    if not roadnet_file_path.startswith("/"):
+        roadnet_file_root_dir = "replay/"
+    
+    if log_file_path.startswith("/"):
+        log_file_root_dir = "replay"
+    if not log_file_path.startswith("/"):
+        log_file_root_dir = "replay/"
```

Comment:
Please simplify the code to
```suggestion
roadnet_file_root_dir = "replay" if roadnet_file_path.startswith("/") else "replay/"
log_file_root_dir = "replay" if log_file_path.startswith("/") else "replay/"
```